atomicfu/src/nativeTest/kotlin/kotlinx/atomicfu/test/SynchronizedTest.kt

```diff
@@ -0,0 +1,38 @@
+package kotlinx.atomicfu.test
+
+import kotlinx.atomicfu.*
+import kotlin.native.concurrent.*
+import kotlin.native.concurrent.AtomicInt
+import kotlin.test.*
+
+class SynchronizedTest {
+
+    private val iterations = 1000
+    private val COUNT = 20
+
+    @Test
+    fun stressCounterTest() {
+        repeat(iterations) { i ->
+            val workers = Array(COUNT) { Worker.start() }
+            val counter = AtomicInt(0).freeze()
+            val so = SynchronizedObject().freeze()
+            workers.forEach { worker ->
+                worker.execute(TransferMode.SAFE, {
+                    counter to so
+                }) {(count, lock) ->
+                    repeat(1000) {
```

Comment:
`1000` needs its own constant, since it is used below in `COUNT * 1000` formula for expected value of counter